### DISCUSSION TITLE
Slideshow

### DISCUSSION THREAD

**Discussion by tsokar** [marp-team/marp#320] [Q&A] [answered]
Hi !
First of all, thanks a lot for the time you devote to marp !

I was wondering if it was possible to create a slideshow with marp (switching to the next slide after a given timing, and possibly looping after the last slide).
I tried to look in the documentation, but I was not able to find any answer.

Thanks !

**Comment by yhatt** [comment, marked as answer]
There is no abiility to make auto navigation and loop in official Marp tools for now. You have to create your own slideshow HTML from rendered snippets by [Marp Core](https://github.com/marp-team/marp-core) manually.

**Reply by tsokar** [reply]
OK, thanks a lot for your reply !
